web/src/settings_invites.ts

```diff
@@ -259,6 +274,8 @@ export function on_load_success(
                 ? $t_html({defaultMessage: "Revoke invitation link"})
                 : $t_html({defaultMessage: "Revoke invitation to {email}"}, {email}),
             html_body,
+            id: "revoke_invite_modal",
+            close_on_submit: false,
```

Comment:
Would be good to pass `loading_spinner: true` as well since modal is opened while the request is being processed.

Reply:
And then we would also not need the "Working" text shown in the resend and revoke button -
![image](https://github.com/user-attachments/assets/bec873ad-261a-45e3-ae5f-b39e723926e3)

Reply:
Okay, so should I remove the code that sets the "Working..." text to the resend and revoke button?
For example, the following line of code, sets the "Working..." text to revoke button:
```
$revoke_button.prop("disabled", true).text($t({defaultMessage: "Working…"}));
```